core/aws-lsp-core/src/index.ts

```diff
@@ -14,3 +14,4 @@ export * as textUtils from './util/text'
 export * as timeoutUtils from './util/timeoutUtils'
 export * from './util/awsError'
 export * from './base/index'
+export * from './test/testFolder'
```

Comment:
Is it not likely that all these non-aliased `*` exports will eventually conflict? And that would then require quasi-namespacing. Instead of quasi-namespacing, we can solve the problem easily by aliasing, like this:

```suggestion
export * as testFolder from './test/testFolder'
```